README.markdown

```diff
@@ -191,7 +171,8 @@ One of the easiest ways to take advantage of this module is to build a single in
 
 ## A Complete Demo Stack
```

Comment:
I'd suggest removing this section rather than linking to something that's out-of-date.